pkg/connector/connector.go

```diff
@@ -378,6 +383,8 @@ func (c *AWS) ResourceSyncers(ctx context.Context) []connectorbuilder.ResourceSy
 		orgClient, err := c.orgClient(ctx)
 		if err == nil {
 			rs = append(rs, accountBuilder(orgClient, c.roleARN, ssoAdminClient, ix, c.ssoRegion, identityStoreClient))
+		} else {
```

Comment:
The other calls are in this style:
```go
orgClient, err := c.orgClient(ctx)
if err != nil {
  l.Error("accountBuilder error", zap.Error(err))
  return rs
}
rs = append(rs, accountBuilder(orgClient, c.roleARN, ssoAdminClient, ix, c.ssoRegion, identityStoreClient))
```
Is this style a Go thing or does it not actually matter?

Reply:
It doesn't really matter, but `if err != nil {` is all over go code because the typical behavior is to early return (as is the case with the other error checks in this function). In this case, we're not returning early so I did the minimum change.

Also if I have an if and an else, and the code inside each is about the same, I tend to use `==` instead of `!=` to improve readability.